setup.py

```diff
@@ -9,13 +9,13 @@
 
 setup(
     name='django-freeipa-auth',
-    version='1.0.0',
+    version='2.0.0',
```

Comment:
I like that you made this a major version change, since it's not backwards compatible with the previous version.